lib/dry/events/publisher.rb

```diff
@@ -236,6 +245,20 @@ def process(event_id, payload = EMPTY_HASH, &block)
         def __bus__
           @__bus__ ||= self.class.new_bus
         end
+
+        # Utility method that check that the event has been registered
+        #
+        # @return [Boolean]
+        #
+        # @api private
+        def event_registered?(object_or_event_id)
+          case object_or_event_id
+          when String, Symbol
+            __bus__.event_registered?(object_or_event_id)
+          else
+            true
```

Comment:
You can `subscribe` to events or you could `subscribe` a listener object, in that case, there is no event registered in the `_bus` to check.

maybe it would be more clear if the code would be:
```ruby
case object_or_event_id
when String, Symbol
  __bus__.event_registered?(object_or_event_id)
when Class
  true
end
```